### PR TITLE
docs: improving headings

### PR DESCRIPTION
**Pull request by peknis**
Leaving out title case and some formatting from headings

Signed-off-by: Pekka Niskanen <pekka.niskanen@nordicsemi.no>

### CODE REVIEW

File: CODE_OF_CONDUCT.rst [hunk 1]
```diff
@@ -90,7 +90,7 @@ Violating these terms may lead to a permanent ban.
 Attribution
 ***********
 
-This Code of Conduct is adapted from the `Contributor Covenant <https://www.contributor-covenant.org>`_, version 2.0, available at https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
+This Code of conduct is adapted from the `Contributor Covenant <https://www.contributor-covenant.org>`_, version 2.0, available at https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
```
Comment (coderbyheart) [inline CODE_OF_CONDUCT.rst:93]: Why keep "Code" capitalized, but not "Conduct"?

File: docs/guides/Direnv.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .. _about-direnv:
 
-direnv
-######
+Managing environment variables
+##############################
```
Comment (coderbyheart) [inline docs/guides/Direnv.rst]:
```suggestion
Managing environment variables using direnv
###########################################
```

File: CODE_OF_CONDUCT.rst (hunk 1 above)
Comment (peknis) [reply CODE_OF_CONDUCT.rst:93]: Good point. I did think about it and thought one might consider it a proper noun, thus capitalizing the first word.

File: docs/azure/GettingStarted/AzureCLI.rst [hunk 3]
```diff
@@ -9,7 +9,7 @@ After installing the CLI, you must be able to execute the ``az`` command:
 
     az
 
-Dockerizing the ``az`` command
+Dockerizing the az command
 ==============================
```
Comment (umapraseeda) [inline docs/azure/GettingStarted/AzureCLI.rst]:
```suggestion
Dockerizing the az command
==========================
```

File: docs/guides/Direnv.rst [hunk 4]
```diff
@@ -1,7 +1,7 @@
 .. _about-direnv:
 
-direnv
-######
+Managing environment variables using direnv
+##############################
```
Comment (coderbyheart) [inline docs/guides/Direnv.rst]:
```suggestion
##############################
```
```suggestion
Managing environment variables using direnv
###########################################
```

**Review by coderbyheart — APPROVED** [review]

**Pull request merged**